builder/vmware/common/driver_config.go

```diff
@@ -22,9 +24,12 @@ type DriverConfig struct {
 	// The type of remote hypervisor that will be used. If set, the remote
 	// hypervisor will be used for the build. If not set, a local desktop
 	// hypervisor (VMware Fusion or VMware Workstation) will be used.
-	// Available options include `esx5` for VMware ESXi.
+	// Available options include `esxi` and `esx5` for VMware ESXi.
+	//
+	// ~> **Note:** Use of `esxi` is recommended; `esx5` is deprecated.
```

Comment:
It may be worth mentioning that `esx5` is now `esxi`?

Reply:
Note: maybe not after reading the PR, it's essentially the same thing, `esx5` is only for backwards compatibility I imagine?
I wonder if, to minimise the diff, we could replace `esx5` with `esxi` during `Prepare` so that all around the code we don't need to evaluate `if RemoteType == "esxi" || RemoteType == "esx5"`, instead we can simplify with `if RemoteType == "esxi"`.
If `esxi` is the only choice we could even make it a `if RemoteType != ""`?

Reply:
Addressed in b9b4318. 😄